lib/DataBase/schema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE
+  music_albums (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    genre VARCHAR(255),
```

Comment:
Kindly remove this `genre` because you have already added `genre_id` as a foreign key referencing the `genres` table and we don't need `genre` as a `VARCHAR` here.